lighthergm/src/run_EM_algorithm.cpp

```diff
@@ -2,10 +2,15 @@
 // BinaryReciprocityModel.cpp for version written already,
 // MMBinaryReciprocityModel.cpp for more eleborate version
 // #define ARMA_64BIT_WORD 1;
-#include <omp.h>
+#ifdef _OPENMP
```

Comment:
@komatsuna4747 just to check, what is exactly the issue here?

Reply:
@justinian336 

The original code yielded `'omp.h' file not found` on Mac. In fact, I don't exactly understand why this modification has solved the issue.

```
── R CMD build ─────────────────────────────────────────────────────────────────
pdflatex not found! Not building PDF manual.
* checking for file ‘.../DESCRIPTION’ ... OK
* preparing ‘lighthergm’:
* checking DESCRIPTION meta-information ... OK
* cleaning src
* installing the package to build vignettes
      -----------------------------------
* installing *source* package ‘lighthergm’ ...
** using staged installation
** libs
clang++ -mmacosx-version-min=10.13 -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/Users/runner/work/_temp/Library/Rcpp/include' -I'/Users/runner/work/_temp/Library/RcppArmadillo/include' -I/usr/local/include  -DARMA_64BIT_WORD=1  -fPIC  -Wall -g -O2  -Wall -pedantic -fdiagnostics-color=always -c RcppExports.cpp -o RcppExports.o
clang++ -mmacosx-version-min=10.13 -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/Users/runner/work/_temp/Library/Rcpp/include' -I'/Users/runner/work/_temp/Library/RcppArmadillo/include' -I/usr/local/include  -DARMA_64BIT_WORD=1  -fPIC  -Wall -g -O2  -Wall -pedantic -fdiagnostics-color=always -c compute_eigenvector.cpp -o compute_eigenvector.o
clang++ -mmacosx-version-min=10.13 -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/Users/runner/work/_temp/Library/Rcpp/include' -I'/Users/runner/work/_temp/Library/RcppArmadillo/include' -I/usr/local/include  -DARMA_64BIT_WORD=1  -fPIC  -Wall -g -O2  -Wall -pedantic -fdiagnostics-color=always -c compute_yule_coef.cpp -o compute_yule_coef.o
clang++ -mmacosx-version-min=10.13 -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/Users/runner/work/_temp/Library/Rcpp/include' -I'/Users/runner/work/_temp/Library/RcppArmadillo/include' -I/usr/local/include  -DARMA_64BIT_WORD=1  -fPIC  -Wall -g -O2  -Wall -pedantic -fdiagnostics-color=always -c create_feature_matrix.cpp -o create_feature_matrix.o
create_feature_matrix.cpp:5:10: fatal error: 'omp.h' file not found
#include <omp.h>
         ^~~~~~~
```